assets/prototype/domain/eventEditor/tickets/ticketPriceCalculator/formDecorators/useReverseCalculateDecorator.js

```diff
@@ -0,0 +1,26 @@
+import { boolsEqual, getFromFormData } from './utilities';
+import useTicketPriceCalculator from '../hooks/useTicketPriceCalculator';
+
+const useReverseCalculateDecorator = () => {
+	const calculator = useTicketPriceCalculator();
+	return {
+		field: 'ticket.reverseCalculate',
+		isEqual: boolsEqual,
+		updates: (value, name, formData) => {
+			const reverseCalc = getFromFormData('ticket.reverseCalculate', formData);
+			return reverseCalc
+				? calculator({
+						data: formData,
+						path: 'prices[0].amount',
```

Comment:
Not sure what is the first element in `prices` array, maybe extract that into a variable with some context?

Reply:
that's not an array there though... that's a string showing the path into the form data the calculator needs to take (it ultimately gets used by Ramda's `path` function).

if I were to extract the form data here, then I would have to refactor the calculator to accept all kinds of different values, which would likely result in not having a single calculator using a reducer, etc.